params/test.yml

```diff
@@ -0,0 +1,8 @@
+genomeIds:
+  - "ERR037481"
+  - "ERR216945"
+  - "ERR1193883"
+
+outdir: "./test_output"
+
+ncbi_api_key: "b6d95ddb1bbd179bf9408c2726dba5ddfe09"
```

Comment:
🤦

Reply:
I always forgot to remove my api keys before commiting :sob: I'll remove it from ncbi